jni/minicap/minicap.cpp

```diff
@@ -506,20 +488,7 @@ main(int argc, char* argv[]) {
 
       haveFrame = true;
 
-      // Encode the frame.
-      if (!encoder.encode(&frame, quality)) {
-        MCERROR("Unable to encode frame");
-        goto disaster;
-      }
-
-      // Push it out synchronously because it's fast and we don't care
-      // about other clients.
-      unsigned char* data = encoder.getEncodedData() - 4;
-      size_t size = encoder.getEncodedSize();
-
-      putUInt32LE(data, size);
-
-      if (pumps(fd, data, size + 4) < 0) {
+      if (pumps(fd, (unsigned char*)frame.data, frame.size) < 0) {
```

Comment:
Can we try and log frame.size here  ? It will help in debugging and knowing if we are sending frames of correct size that client is expecting. 
Obv we should remove/comment the log since we dont want to take cpu cycles to log in this code path as we want to keep it as performant as possible.